init.lua

```diff
@@ -201,6 +259,26 @@ minetest.register_abm( {
 	end
 } )
 
+minetest.register_lbm({
```

Comment:
Why exactly is this LBM needed? Would you please be so nice to add a comment in the code above?